@xen-orchestra/web-core/lib/components/fullscreenable/FullscreenableHeader.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <header class="header">
+    <UiIcon :icon="faAngleLeft" color="info" @click="back" />
+    <slot />
+  </header>
+</template>
+
+<script lang="ts" setup>
+import UiIcon from '@core/components/icon/UiIcon.vue'
+import { faAngleLeft } from '@fortawesome/free-solid-svg-icons'
+
+defineProps<{
+  back: () => void
+}>()
```

Comment:
I'd use events instead:

```suggestion
const emit = defineEmits<{
  close: []
}>()
```

Then replace `@click="back"` with `@click="emit('close')"` above.